models/cice/shell_scripts/CESM2_3_setup_ensemble

```diff
@@ -0,0 +1,383 @@
+
+EndOfText
+
+   #--------------------------------------------#
+   # user_nl_cices
+   #--------------------------------------------#
+   set ice_fname = "user_nl_cice_${inst_string}"
+   if ( $runtype != "branch" || $runtype != "hybrid" ) then
+      if ( $refcaseens == "false" ) then
+         echo "ice_ic = '${refcasedir}/${refcase}.cice.r.${refdate}-00000.nc'" >> $ice_fname
+      else
+         echo "ice_ic = '${refcasedir}/${refcase}.cice_${inst_string}.r.${refdate}-00000.nc'" >> $ice_fname
+      endif
+   endif
+
+   echo "r_snw = 1.5 "                         >> $ice_fname
+   echo "dt_mlt = 1.0 "                        >> $ice_fname
+   echo "histfreq_n =  1,1,1,1,1  "            >> $ice_fname
+   echo "histfreq   = 'd','m','x','x','x' "    >> $ice_fname
+   echo "f_sst = 'dmxxx' "                     >> $ice_fname
+   echo "f_sss = 'dmxxx' "                     >> $ice_fname
+   echo "f_frzmlt = 'dmxxx' "                  >> $ice_fname
+   echo "f_frz_onset = 'dmxxx' "               >> $ice_fname
+   echo "f_aicen = 'dmxxx' "                   >> $ice_fname
+   echo "f_vicen = 'dmxxx' "                   >> $ice_fname
+   echo "f_vsnon = 'dmxxx' "                   >> $ice_fname
+   echo "f_hi = 'dmxxx' "                      >> $ice_fname
+   echo "f_hs = 'dmxxx' "                      >> $ice_fname
+   echo "f_aice = 'dmxxx' "                    >> $ice_fname
+   echo "f_snowfrac = 'dmxxx' "                >> $ice_fname
+   echo "f_albsni = 'dmxxx' "                  >> $ice_fname
+   echo "f_albsno = 'dmxxx' "                  >> $ice_fname
+   echo "f_albice = 'dmxxx' "                  >> $ice_fname
+   echo "f_albpnd = 'dmxxx' "                  >> $ice_fname
+   echo "highfreq = .false. "                  >> $ice_fname
+
+   #--------------------------------------------#
+   # user_nl_drofs
+   #--------------------------------------------#
+   set rof_fname = "user_nl_drof_streams_${inst_string}"
+
+   cat << EndofText >> $rof_fname
+   rof.diatren_iaf_rx1: year_first=$stream_year_first 
+   rof.diatren_iaf_rx1: year_last=$stream_year_last
+
+EndofText
+
+   #--------------------------------------------#
+   # user_nl_docns
+   #--------------------------------------------#
+   set ocn_fname = "user_nl_docn_streams_${inst_string}"
+   cat << EndofText >> $ocn_fname
+
+   som: datafiles = /glade/p/cesmdata/cseg/inputdata/ocn/docn7/SOM/pop_frc.b.e21.BW1850.f09_g17.CMIP6-piControl.001.190514.nc
+   som: meshfile = /glade/p/cesmdata/cseg/inputdata/share/meshes/gx1v7_151008_ESMFmesh.nc
+
+EndofText
+
+
+   @ inst = $inst + 1
+end
+
+./preview_namelists
+
+# ==============================================================================
+# Introduce SourceMods, if any
```

Comment:
It does not look like the source mods are setup by this script. 
Are you doing a cp of the SourceMods into your case directory outside this setup script?